src/DocumenterVitepress.jl

```diff
@@ -88,4 +88,30 @@ function Documenter.postprocess_before_push(versions::BaseVersion; subfolder, de
     return
 end
 
+function deploydocs(;
+    repo,
+    build_dir,
+    branch,
+    devbranch,
+    push_preview,
+)
+    bases = readlines(joinpath(build_dir, "bases.txt"))
+    @info "Found bases for deployment: $bases"
+
+    for (i, base) in enumerate(bases)
+        @info "Deploying docs for base $(repr(base))"
+        dir = joinpath(build_dir, "$i")
+        Documenter.deploydocs(;
+            repo,
+            target = dir, # each version built has its own dir
+            versions = DocumenterVitepress.BaseVersion(base),
+            dirname = base,
+            branch,
+            devbranch,
+            push_preview,
+        )
+    end
+
+end
```

Comment:
Can this be `AutoBaseVersions(builddir = "build")` instead?  That way this is as unintrusive as possible, easier to switch back and forth as well.

Reply:
Hm yeah but `versions` is not enough to handle the multi deploy logic. In that case I guess you'd have to add a positional argument dispatch.

Reply:
I have changed `build_dir` to `target` so now it looks like the normal `deploydocs` syntax, just with `DocumenterVitepress.` prepended which I think is sufficiently straightforward